ts/parts/Time.ts

```diff
@@ -152,7 +152,7 @@ declare global {
  * @callback Highcharts.TimezoneOffsetCallbackFunction
  *
  * @param {number} timestamp
- * Timestamp in terms of milliseconds since January 1 1970.
+ * Date instance or timestamp in terms of milliseconds since January 1 1970.
```

Comment:
Since you changed the occurences in the code to return `.valueOf()`, is this always a timestamp now?

Reply:
Thanks, good catch! Yes, this doclet change is therefor invalid.